debian/wott-agent-self-update.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=wott-agent self update
+
+[Service]
+Type=oneshot
+ExecStart=/opt/wott/wott-agent-self-update.sh
```

Comment:
Let's create a `bin` directory under `/opt/wott` as we will likely need more scripts in the future.